index.html

```diff
@@ -2458,14 +2458,24 @@ <h3>Data Schemas</h3>
 
         <dl>
           <dt><var>credentialSchema</var></dt>
-          <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
-more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
-<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
-that MUST be a <a>URL</a> identifying the schema file. The precise contents of
-each data schema is determined by the specific type definition.
+          <dd><p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be a single
+data schema object which provides <a>verifiers</a> with enough information 
+to determine if the provided data conforms to the provided schema object. Each
+<code>credentialSchema</code> MUST specify its <code>type</code> (e.g.,
+<code>JsonSchemaValidator2018</code>, 
+<code><a href="https://w3c-ccg.github.io/vc-json-schemas/">CredentialSchema2022</a></code>,
+or <code><a href="https://transmute-industries.github.io/vc-credential-schema-open-api-specification/">
+OpenApiSpecificationValidator2022</a></code>), and an <code>id</code> <a>property</a>
```

Comment:
We shouldn't link to vendor organization Github repos in standards-track specs because 1) not all vendor organizations stick around for as long as these specs do and the links break over time, and 2) competing vendors complain of low-key favoritism when we do this.

Ideally, all external links should point to stable documents that we expect to last for a decade or more.

Reply:
The other reference is to the W3C CCG which also relies on "GitHub" for hosting.

Reply:
Additional context, there are at least 4 companies using this approach today, its the same approach used by 
- https://github.com/w3c-ccg/traceability-vocab

Reply:
@OR13 can you move it into the CCG and I'll add it back then?

Reply:
@decentralgabe we are not required to "move things to CCG" by W3C Process. 

We can adopt them directly as work items, as a working group.

I suggest adding a note saying it will be removed in future draft if its not adopted as a work item, to both links.

Reply:
Waiting for you and @msporny to have consensus before proceeding here